tests/comments_jsdoc_typescript/type.js

```diff
@@ -0,0 +1,8 @@
+const foo1 = /** @type {!Foo} */(bar);
```

Comment:
Please rename `comments_jsdoc_typescript/type.js` → `comments_closure_typecast/extra_star.js`

Reply:
Here extra space too, maybe `extra_spaces_and_asterisks.js`

Reply:
ok

Reply:
I think better name is `type-casting-with-extra-spaces-and-asterisks.js` (long, but reflects everything)

Reply:
The folder is already named `comments_closure_typecast`.

Reply:
oh, yes, my mistake

Reply:
fixed